INTERPRETING.md

```diff
@@ -93,6 +93,7 @@ properties of the global scope prior to test execution.
         sleeps the agent for approximately that duration.
       - **`leaving`** - a function that signals that the agent is done and
         may be terminated (if possible).
+      - **`monotonicNow`** - a function that returns a value that conforms to `DOMHighResTimeStamp` and is produced in such a way that its semantics conform to **Monotonic Clock**.
```

Comment:
From https://www.w3.org/TR/hr-time-2/#sec-domhighrestimestamp

> A DOMHighResTimeStamp SHOULD represent a time in milliseconds accurate enough to allow measurement while preventing timing attack

Should we ask for an even more precise clock when it's available?

Reply:
maybe link to this part: https://www.w3.org/TR/hr-time-2/#x7-monotonic-clock

Reply:
Done